src/app/components_ngrx/group-types-panel/group-types-panel.component.html

```diff
@@ -3,9 +3,11 @@
     class="f8-group-filter__type"
     id="{{item.group}}"
     [routerLink]="[]"
-    [queryParams]="showTree ?
-      {q: fnBuildQueryParam(item), showTree: showTree} :
-      {q: fnBuildQueryParam(item)}"
+    [queryParams]="showTree && showCompleted ?
```

Comment:
Why is it named ```showCompleted```, not ```showClosed```?

Reply:
This comes up after Todd and Nimisha discussion on visual design. @nimishamukherjee please confirm.